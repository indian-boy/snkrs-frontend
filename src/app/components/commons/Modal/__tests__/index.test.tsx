import { render } from '@testing-library/react';
import * as React from 'react';
import { Modal } from '..';

const MockComponent = () => {
  return <div>Modal</div>;
};

const MockedComponenteWithModal = Modal(MockComponent);

describe('<MockedComponenteWithModal  />', () => {
  it('should match snapshot', () => {
    const mockedComponenteWithModal = render(<MockedComponenteWithModal />);
    expect(mockedComponenteWithModal.container.firstChild).toMatchSnapshot();
  });
});
