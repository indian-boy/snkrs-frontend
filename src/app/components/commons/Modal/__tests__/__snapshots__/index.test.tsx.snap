// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<MockedComponenteWithModal  /> should match snapshot 1`] = `
.c0 {
  position: fixed;
  z-index: 999;
  left: 5%;
  top: 10%;
  width: 90%;
  height: 80%;
  overflow: auto;
  box-shadow: 0 0 4rem rgb(0 0 0 / 10%);
  background-color: #FFFFFF;
  border-radius: 0.625rem;
}

.c1 {
  position: fixed;
  right: 5%;
  bottom: 3%;
  background-color: transparent;
  cursor: pointer;
  outline: none;
  text-align: center;
  -webkit-user-select: none;
  -moz-user-select: none;
  -ms-user-select: none;
  user-select: none;
  display: -webkit-inline-box;
  display: -webkit-inline-flex;
  display: -ms-inline-flexbox;
  display: inline-flex;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  border: none;
  padding: 0.5rem;
}

<div
  class="c0"
>
  <button
    class="c1"
  >
    <svg>
      close.svg
    </svg>
  </button>
  <div
    data-testid="modalID"
  >
    Modal Content
  </div>
</div>
`;
